server/scripts/start-test-server.sh

```diff
@@ -48,12 +50,23 @@ check_containers_health() {
   all_healthy=true
 
   for container in $(docker ps -q); do
+    health_field=$(docker inspect --format='{{.State.Health}}' "$container")
```

Comment:
If during test some other containers are running - check if health status is not defined - skip container information.